apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/site-title/edit.js

```diff
@@ -0,0 +1,99 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import React, { Component } from 'react';
+import { __ } from '@wordpress/i18n';
+const { withSelect } = wp.data;
+const { apiFetch } = wp;
+
+/**
+ * Internal dependencies
+ */
+
+class SiteTitleEdit extends Component {
+	constructor( props ) {
+		super( props );
+
+		this.state = {
+			title: null,
+			initialTitle: null,
+		};
+	}
```

Comment:
No need for the constructor here. It's more common for binding functions to the right `this`, but we can also use fat arrow in those related functions. eg `clickOnButton = () => {}` 
```
state = {
    title: null,
    initialTitle: null,
}
```